RevenueCatUI/CustomerCenter/Views/CreateTicketView.swift

```diff
@@ -0,0 +1,169 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  CreateTicketView.swift
+//
+//  Created by Rosie Watson on 11/10/2025
+
+@_spi(Internal) import RevenueCat
+import SwiftUI
+
+#if os(iOS)
+
+@available(iOS 15.0, *)
+@available(macOS, unavailable)
+@available(tvOS, unavailable)
+@available(watchOS, unavailable)
+struct CreateTicketView: View {
+
+    @Environment(\.appearance)
+    private var appearance: CustomerCenterConfigData.Appearance
+
+    @Environment(\.colorScheme)
+    private var colorScheme
+
+    @Environment(\.localization)
+    private var localization: CustomerCenterConfigData.Localization
+
+    @Environment(\.navigationOptions)
+    var navigationOptions
+
+    @Binding
+    private var isPresented: Bool
+
+    @State
+    private var email: String = ""
+
+    @State
+    private var description: String = ""
+
+    @State
+    private var isSubmitting: Bool = false
+
+    @State
+    private var errorMessage: String?
+
+    private let purchasesProvider: CustomerCenterPurchasesType
+
+    init(isPresented: Binding<Bool>, purchasesProvider: CustomerCenterPurchasesType) {
+        self._isPresented = isPresented
+        self.purchasesProvider = purchasesProvider
+    }
+
+    var body: some View {
+        CompatibilityNavigationStack {
+            Form {
+                Section(header: Text(localization[.email])) {
+                    TextField(localization[.enterEmail], text: $email)
+                        .keyboardType(.emailAddress)
+                        .autocapitalization(.none)
+                        .textContentType(.emailAddress)
+                }
+
+                Section(header: Text(localization[.description])) {
+                    TextEditor(text: $description)
+                        .frame(minHeight: 150)
+                }
+
+                Section {
+                    Button(action: submitTicket) {
+                        HStack {
+                            Spacer()
+                            if isSubmitting {
+                                ProgressView()
+                                    .progressViewStyle(CircularProgressViewStyle())
+                            } else {
+                                Text(localization[.submitTicket])
+                                    .font(.headline)
+                            }
+                            Spacer()
+                        }
+                    }
+                    .disabled(email.isEmpty || description.isEmpty || isSubmitting)
```

Comment:
We do have some email validation in Android, we should probably add that here as well